models/survey.py

```diff
@@ -0,0 +1,52 @@
+import datetime
+
+from app import app, db
+from flask import flash, render_template, request, redirect
+from wtforms import Form, IntegerField, StringField
+
+class Survey:
+    __tablename__ = "surveys"
+ 
+    id = int()
+    description = str()
+    start_date = datetime.date
+    end_date = datetime.date
+
+    def __init__(self, id_, description_):
+        self.id = id_
+        self.description = description_
+
+    def __repr__(self):
+        return "<Question: {}>".format(self.name)
+
+class surveyForm(Form):
+    id = IntegerField("ID")
+    description = StringField("Description")
+
+@app.route('/addNewSurvey', methods=['GET', 'POST'])
```

Comment:
These routes and their underlying methods deserve a directory of their own. Try to move all routes and their functions to `/controllers` path. I believe these are just for testing cause they do **NOT** follow the specs stated on Trello. Take a look at "Create CRUD for Survey" [here](https://trello.com/c/gJl3omU7).

Reply:
Yes, they are just a test for IO :)! I'll try it out with controllers. Is there some common practice on naming controller-classes and files?

Reply:
@PetNamedEr I guessed 😉 Well, controllers should be named as plural nouns. Like "Surveys" or "Questions". Though models on the other hand should be named as singular nouns.

Reply:
I made controllers/surveys.py